src/fmu/dataio/export/_enums.py

```diff
@@ -43,11 +43,35 @@ def index_columns() -> list[str]:
         """Returns a list of the index columns."""
         return [k.value for k in InplaceVolumes.TableIndexColumns]
 
+    @staticmethod
+    def required_index_columns() -> list[str]:
+        return [
+            InplaceVolumes.TableIndexColumns.FLUID.value,
+            InplaceVolumes.TableIndexColumns.ZONE.value,
+        ]
+
     @staticmethod
     def value_columns() -> list[str]:
         """Returns a list of the value columns."""
         return [k.value for k in InplaceVolumes.VolumetricColumns]
 
+    @staticmethod
+    def required_value_columns() -> list[str]:
+        """Returns a list of the value columns."""
+        return [
+            InplaceVolumes.VolumetricColumns.BULK.value,
+            InplaceVolumes.VolumetricColumns.PORV.value,
+            InplaceVolumes.VolumetricColumns.HCPV.value,
+        ]
+
+    @staticmethod
+    def required_columns() -> list[str]:
+        """Returns a list of the columns required at export."""
+        return (
+            InplaceVolumes.required_index_columns()
+            + InplaceVolumes.required_value_columns()
+        )
```

Comment:
This is becoming a kind of verbose "enum". I think it's OK for now, but I can see that the current module/subpackage layout will need adjustment for the next one. It's probably a bit too early to see how that should look now